deploy/kubeturbo-operator/test.sh

```diff
@@ -2,48 +2,56 @@
 
 SCRIPT_DIR="$(cd "$(dirname $0)" && pwd)"
 ERR_LOG=$(mktemp --suffix _kube.errlog)
-WAIT_FOR_DEPLOYMENT=20
+WAIT_FOR_DEPLOYMENT=30
 OPERATOR_IMAGE_VERSION="8.9.5-SNAPSHOT"
 OPERATOR_IMAGE="icr.io\/cpopen\/kubeturbo-operator:${OPERATOR_IMAGE_VERSION}"
+KUBETURBO_IMAGE_VERSION="8.9.5-SNAPSHOT"
+KUBETURBO_IMAGE_REPO="icr.io/cpopen/turbonomic/kubeturbo"
+KUBECTL=kubectl
+# username and password for the local ops-manager
+OPS_MANAGER_USERNAME=administrator
+OPS_MANAGER_PASSWORD=administrator
 
 # turbonomic is the default namespace that matches the xl setup
 # please change the value according to your set up
 export namespace=turbonomic
 
+function rediscover_target {
+	DISPLAY_NAME=$1
+    ip=$($KUBECTL get services --no-headers -n $namespace | grep nginx | awk '{print $4; exit}' | cut -d "," -f 1)
```

Comment:
I don't quite understand here. Why are we trying to grep `nginx` service?  What it it used for? who will deploy it?

Reply:
It's just one of the ways to detect the IP of the instance where I ran my tests instead of hardcoding it. It might need to be updated when @LionelWuW integrates the tests in jenkins.

Reply:
This is the same way that the exported method `xl_version` uses in the `.bashrc` to get the IP address.

Reply:
I understand you're trying to make a broadcast through`nginx` here. But If we have this in here, it means our test will strongly depend on the server component.  Kubeturbo has to be deployed in the same namespace as the other server component. Usually, that's not our deploy model except turbo on turbo.  Moreover, I'm not sure if the DevOps team can help with that when they run this test in the Jenkins pipeline.

Reply:
I am using to the target API to rediscover the target, so this method will depend on the server regardless. Is there another way to trigger rediscovery of target without going through the server? 

btw, for the testcases, we are using deploying kubeturbo operator and kubeturbo in separate ns than the server component.

Reply:
I'm thinking about another way to verify this instead of a broadcast. What I have in my mind is we can kill the kubeturbo which will trigger the following message.  If we take it this way, I guess we don't need to involve a server component.  Does this make sense?
```
glog.V(4).Infof("*** Handling Kubeturbo Termination ***")
```

Reply:
After discussion, I removed the rediscover logic so there's no dependency on XL server.